search-sentiments-map/public/scripts/current-data-script.js

```diff
@@ -12,39 +12,42 @@
 // License for the specific language governing permissions and limitations under
 // the License.
 
+// String for the topic that the user is currently viewing.
```

Comment:
Just saying topic.... is enough. (I.e. remove the "String" part)

Reply:
ok